### ISSUE TITLE
Catch badly put newlines in LESS files.

### ISSUE THREAD

**Issue by svera** [magento/magento-coding-standard#242]
### Description

Newlines existing in the middle of a style definition for LESS files should be catch. Currently, `Magento2/Sniffs/Less/ColonSpacingSniff.php` has this checking in place but it is not working, as PHPCS strips newlines that goes in the middle of a sentence.
### Expected behavior

Being able to catch the following case
```css
.foo {
  bar:
          'xyz';
}
```
### Benefits

An incorrect LESS formatting would be catch
